js/modules/k6/experimental/csv/module.go

```diff
@@ -0,0 +1,238 @@
+// Package csv provides a CSV parser for k6.
+package csv
+
+import (
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"io"
+	"sync/atomic"
+
+	"gopkg.in/guregu/null.v3"
+
+	"go.k6.io/k6/js/promises"
+
+	"go.k6.io/k6/js/modules/k6/experimental/fs"
+
+	"github.com/dop251/goja"
+	"go.k6.io/k6/js/common"
+	"go.k6.io/k6/js/modules"
+)
+
+// TODO: Should we have an option to cycle through the file (restart from start when EOF is reached, or done is true)?
+// TODO: Should we have an option to skip empty lines (lines with no fields?)
```

Comment:
My first reaction is: why not doing so by default? But I guess we should/could take a look at similar parser libraries/tools, and get inspiration from there (what looks to be more common).

PS: Do you mean lines with at least one value missing, or those lines completely empty? 🤔

Reply:
Regarding `skipEmptyLines`, it turns out that skipping them is the default behavior of the Go CSV parser, and it looks like it can't be overridden, so it sorts it out 👍🏻